README.md

```diff
@@ -147,6 +147,35 @@ in `rustfmt.toml`.
 [Rust](https://github.com/rust-lang/rust) code.
 - Alternatively, run `make format-lint` to run both at the same time.
 
+# Run Benchmarking
+
+In order to achieve a certain block time, for example 6 seconds per block,
+there is a limited number of lines of code can be run during that time frame.
+When writing a pallet function, the developer is responsible for calculating its
+computational complexity, which is called **weight**. The process of
+determining that complexity, or simply put, time cost is called benchmarking.
+
+1. Build your Frequency node binary with the `runtime-benchmarks` feature flag:
+
+   ```sh
+   cargo build --release --features runtime-benchmarks
+   ```
+1. Execute the benchmarks using `pallet_msa` as an example:
```

Comment:
These instructions aren't completely accurate; for initial generation of benchmarks, using --dev for the chain parameter is fine, however we're using a different chain spec for running benchmarks on the reference hardware, and for external pallets such as orml_vesting and scheduler, we use a different template, `runtime-weight-template.hbs`

Reply:
It should be mainnet (aka `frequency`) once #352 is merged.